### ISSUE TITLE
Not working at all

### ISSUE THREAD

**Issue by MichalLytek** [KamoHaladus/VSC-Hakaton#1]
I installed the extensions and in `setting.json` set the glob to `**/*.ts` but it doesn't work. So I got back to the default setting (`"ngIntelliSense.componentGlob": "**/*Component.ts",`) and create exactly the example code and it still doesn't work, even after reloading VSCode:

![2016-11-09 20_55_15-ustawienia](https://cloud.githubusercontent.com/assets/10618781/20152662/5fc607ac-a6bf-11e6-8915-e1c50e14aa15.png)

Here is the code:

![2016-11-09 21_02_23-ustawienia](https://cloud.githubusercontent.com/assets/10618781/20152757/dca51826-a6bf-11e6-8255-920d71dc12c3.png)

How could I send you a debug log or sth to let you reproduce the bug?

**Comment by kf-ireneuszpatalas** [comment]
I see what's the problem. Component parser basically takes the entire component configuration object and parses it as a JSON. The thing is comments are not valid in JSON (at least JSON.parse doesn't like them very much) and so it cannot analyze this component. During development we have had few situations that JSON.parse couldn't eat and had to "fix" them before parsing, but multiline comments was not one of them. Long story short.. that's a bug. Thanks for finding this :)
This will be fixed in next version (should be quite soon).

In the meantime try with a real component without comments like:

``` TypeScript
angular.module('app').component('exampleComponent', {
    templateUrl: 'componentView.html',
    controller: 'ComponentController',
    bindings: {
        config: '<',
        data: '<'
    }
});
```

If it still doesn't work then please isolate the problem as much as possible (one component file and one html file where you're trying to use the component would be perfect) and attach such project here. We'll have a look what's the problem.

**Comment by MichalLytek** [comment]
After removing the comment it works (after reloading, so maybe it should watches for file changes and updates intellisense or you could do a statusbar button to refresh the cache?).

But it looks like parsing as JSON is a problem. I use the non-string syntax for controller:

``` typescript
class ComponentController {
    ...
}

angular.module('app').component('exampleComponent', {
    templateUrl: 'componentView.html',
    controller: ComponentController,
    bindings: {
        config: '<',
        data: '<'
    }
});
```

And it stops working. Also the webpack `require` makes it not working:

```
template: require("./example.html")
```

So it looks like you would need a parser for JS not for JSON I think.

**Comment by kf-ireneuszpatalas** [comment]
Good point. We have 270+ components in our project and none of them used that syntax. Putting it to road map. 
For now we only use bindings part of the configuration, so next minor version might just extract these and skip the other properties for current features to work correctly. This will give the stability for now and time to rethink the parsing when other fields will become necessary.

**Comment by kf-ireneuszpatalas** [comment]
Fixed in 0.2.0
